mmcls/datasets/transforms/auto_augment.py

```diff
@@ -7,8 +7,7 @@
 
 import mmcv
 import numpy as np
-from mmcv import BaseTransform, RandomChoice
-from mmcv.transforms import Compose
+from mmcv.transforms import BaseTransform, Compose, RandomChoice
 from mmcv.transforms.utils import cache_randomness
 from mmengine import is_list_of, is_seq_of
```

Comment:
It would be better to modify this line to:
```python
from mmengine.utils import is_list_of, is_seq_of
```

Reply:
I think this line is fine https://github.com/open-mmlab/mmengine/blob/bf5c26538eb977265574e07098c28c8536a07ec9/mmengine/__init__.py#L7